tox.ini

```diff
@@ -33,3 +33,16 @@ deps =
   -r{toxinidir}/requirements-dev.txt
   -r{toxinidir}/docs_guide/requirements-extra.txt
 commands = sphinx-build -b doctest -W --keep-going -j auto {posargs} docs_guide/ docs_guide/_build/doctest
+
+[testenv:fmt]
+deps =
+  -r requirements-dev.txt
+commands =
+  black docs_guide/ example_docs/ qiskit_sphinx_theme/ tests/  setup.py
+  isort docs_guide/ example_docs/ qiskit_sphinx_theme/ tests/ setup.py
+[testenv:lint]
```

Comment:
Nit:

```suggestion
  isort docs_guide/ example_docs/ qiskit_sphinx_theme/ tests/ setup.py

[testenv:lint]
```

Reply:
thank you and this situation are in therepository you gave me, [qiskit-metapackage](https://github.com/Eric-Arellano/qiskit-metapackage)

Reply:
Oh oops, my mistake in qiskit-metapackage.